extensions/wrapper/wrapper/src/test/resources/offering/contract-offer-valid.json

```diff
@@ -0,0 +1,38 @@
+{
+  "assetEntryDto": {
```

Comment:
whether names need to contain dto can be discussed. Types of course can contain Dto to avoid collisions